src/wcmUSB.c

```diff
@@ -464,6 +464,21 @@ static Bool usbWcmInit(InputInfoPtr pInfo, char* id, size_t id_len, float *versi
 		common->wcmResolX = common->wcmResolY = 1016;
 	}
 
+	/* Mark protocol 5 devices which predate the use of Intuos5 tech
+	 * as using legacy IDs. These legacy IDs do not conform to the
+	 * current format, in particular with respect to the use of the
+	 * 0th device ID bit signalling EMR/AES.
+	 */
+	if (common->wcmModel == &usbIntuos || common->wcmModel == &usbIntuos2 ||
+	    common->wcmModel == &usbIntuos3 || common->wcmModel == &usbIntuos4 ||
+	    (sID.vendor == WACOM_VENDOR_ID && sID.product == 0x3F) ||
+	    (sID.vendor == WACOM_VENDOR_ID && sID.product == 0xC5) ||
+	    (sID.vendor == WACOM_VENDOR_ID && sID.product == 0xC6) ||
+	    (sID.vendor == WACOM_VENDOR_ID && sID.product == 0xCC))
```

Comment:
I'd really prefer the ids to be expanded to identifiable names here. Or at least some comment because it's impossible to check whether the code is correct without any intention. Could be a typo in there but we'd never know.

3f is the 21UX, c5 the 20WSX? and so on?

Reply:
Comments added to make the intended devices more explicit.